posts/inside-rust/2024-10-31-compiler-team-reorg.md

```diff
@@ -0,0 +1,243 @@
+- [nnethercote](https://github.com/nnethercote), maintainer
+    - nnethercote has been working on compiler performance since 2016, including the benchmarking
+      and profiling infrastructure. He has also cleaned up a lot of old crufty code across many
+      parts of the compiler.
+- [oli-obk](https://github.com/oli-obk), maintainer
+    - oli-obk is a long-time compiler team member whose prolific contribution history is a
+      challenge to summarize, but include constant evaluation, constant generics, pattern types,
+      MIR optimisations, diagnostics, clippy improvements, and existential types.
+- [petrochenkov](https://github.com/petrochenkov), maintainer
+    - petrochenkov is another long-time compiler team member who primarily maintains the compiler's
+      name resolution and macro expansion, notoriously tricky and nuanced subsystems of the
+      compiler.
+- [pnkfelix](https://github.com/pnkfelix), maintainer
+    - Former compiler team co-lead, pnkfelix is yet another long-time team member has made
+      contributions throughout the compiler and made significant contributions to the borrow
+      checker, early MIR, and early compiler architecture.
+- [RalfJung](https://github.com/ralfjung), team member
+    - Known for his work on Miri and Stacked Borrows, an operational semantics for memory accesses
+      in the language, RalfJ is the team's foremost expert on the operational semantics of the
+      language and also deeply involved in const evaluation. He has been working on achieving sound
+      semantics down to lowest levels of the compiler.
+- [saethlin](https://github.com/saethlin), maintainer
+    - saethlin has made significant improvements to MIR, Miri, and codegen in their contributions
+      since starting in 2021 and has become an invaluable source of knowledge on the language's
+      operational semantics/unsafe code guidelines.
+- [scottmcm](https://github.com/scottmcm), team member
+    - scottmcm is a member of the language team who also regularly implements improvements in the
+      compiler, particularly in the MIR and compiler's codegen, always trying to get to the perfect
+      machine code.
+- [SparrowLii](https://github.com/sparrowlii), maintainer
+    - SparrowLii is a relatively new member of the team who has resurrected and led the effort to
+      parallelize the compiler and has been responsible for the great strides that effort has made
+      alongside members of the parallel rustc and compiler performance working groups.
+- [spastorino](https://github.com/spastorino), maintainer
+    - spastorino is another alum of the non-lexical lifetimes working group, starting to contribute
+      in late 2017. Since NLL, spastorino has implemented negative impls in coherence, refactored
+      return position impl trait in trait code to lower as a GAT, and has done a lot of bugfixes
+      to the compiler in general and with to the type system as a member of the types team.
```

Comment:
```suggestion
- [spastorino](https://github.com/spastorino), maintainer
    - spastorino is another alum of the non-lexical lifetimes working group, starting to contribute
      in late 2017. Since NLL, spastorino has implemented negative impls in coherence, refactored
      return position impl trait in trait code to lower as a GAT, and has done a lot of refactors/bugfixes
      to the compiler.
```